contributing/development/code_style_guidelines.rst

```diff
@@ -35,7 +35,7 @@ fine. If you plan to contribute regularly, however, we strongly advise that you
 set up clang-format locally to check and automatically fix all your commits.
 
 .. warning:: Godot's code style should *not* be applied to third-party code,
-             i.e. that is included in Godot's source tree but was not written
+             i.e. that is included in Godot's source tree, but was not written
```

Comment:
```suggestion
             i.e. that is included in Godot's source tree but was not written
```

Reply:
does this look like a good one to revert too?

Reply:
I'd keep the comma here. Also, I think there should be "code" just after "i.e". It would make the sentence read more naturally (closer to how you'd read it aloud).

Reply:
i'll push up a commit to add the word "code"